awx/api/urls/urls.py

```diff
@@ -153,6 +154,7 @@
     re_path(r'^bulk/$', BulkView.as_view(), name='bulk'),
     re_path(r'^bulk/host_create/$', BulkHostCreateView.as_view(), name='bulk_host_create'),
     re_path(r'^bulk/job_launch/$', BulkJobLaunchView.as_view(), name='bulk_job_launch'),
+    re_path(r'^peers/', include(peers_urls)),
```

Comment:
I don't understand why we are doing top-level 'peers' endpoints.  Peers implies that we are looking in the context of some particular instance, yeah?

Maybe as I get further down I'll see why and this comment will just recommend changing the name to 'links' or something.

Reply:
dropped this endpoint